modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -75,6 +75,15 @@ export default {
     docId: {
       type: String,
       default: null
+    },
+    filterValues: {
+      type: Object,
+      default() {
+        return {
+          published: true,
+          trash: false
+        };
+      }
```

Comment:
@boutell @abea I decided to pass "unpublished" and "trash" filters to the request fetching data in the backend. This is to avoid a bizarre situation where every unpublished or trash status was allowed and tests were failing (or needed to be redesigned).

Reply:
OK, I see you're passing them all in from the pieces manager, so that works. However see my note about redundant req.query code below.

Reply:
@falkodev @boutell Do we need to be passing this through from the manager? If it's a getOne request, so could we always have trash and published set to `null` in the query?